### ISSUE TITLE
[Đà Nẵng] Middle AI Engineer

### ISSUE THREAD

**Issue by LCD-Nhien** [awesome-jobs/vietnam#3979]
## Location

* Da Nang city 

## Job description
* Join in developing and applying the ML/DL techniques to resolve our client’s business problems such as building chatbot systems, LLM/Prompt, OCR systems, fraud detection systems, facial recognition systems…
* Join in developing some internal products by applying AI models

## Must Have

- Have programming skills in one of these languages: Python, Java...
- At least 1 year of experience working with AI/ML projects as a Data Engineer, Research Engineer, or Software Engineer
- Have experience with one of ML/DL Frameworks: Tensorflow, Pytorch, Keras, NumPy, scikit-learn, Pandas, NLTK…
- Have experience with one of chatbot building frameworks or services like Rasa, Spacy, Dialogflow, Transfomer, BERT, LLM/Prompt…
- Familiar with OCR algorithms or services: OpenCV, Tesseract, Textract (AWS), Google Cloud Vision
- Experience in Computer Vision: CNN, YOLO, MTCNN, SSD, Facenet
- Experience with common development tools: Linux, GPU server, Google Colab, Jupyter, Git, Docker
- MUST be good at verbal/non verbal English

**Skills Tags:** AI, Chatbot, NLP, Computer Vision, DB, Git

## NICE TO HAVE

- Experience with backend web development (Flask, Django)
- Experience with SQL database (PostgresSQL, MySQL) and NoSQL database (MongoDB).
- Experience with building APIs and services using REST, SOAP, etc.
- Experience in MLOps system: MLflow, DVC, Feature Store, Vector Database, Airflow, CI/CD... 
- Experience in Generative AI, Large Language Model, Prompt Engineer: Diffusion model, LLaMA, LangChain..

## BENEFITS

* Competitive Salary and Brilliant Health Benefits
- [ ] Attractive salary (13th-month salary, salary review twice/year) and project bonus
- [ ] Bonus programs for candidate referral, technical article writing
- [ ] Interest-free loan support for personal plan 
- [ ] Allowance for sickness, maternity and paternity and Periodic health examination
- [ ] PVI health care program The staff of the quarter and year reward

* Progressive and Fun Working Environment
- [ ] A professional English-speaking working environment with Agile – Scrum model
- [ ] Hybrid Working Model: Flexible working time and WFH support. 
- [ ] Surrounded by friendly, open-minded, young and supportive colleagues Annual company trip and regular team-building parties, party celebrations (Christmas, Birthday, Mid-autumn,...), Sports clubs (football, badminton, swimming …)

* Valuable Training
- [ ] Sponsor examination fee for professional certificates (AWS, Azure, IELTS, PMP, Scrum Master,...)
- [ ] Sponsor fee for joining any technical training sessions and courses.
- [ ] Free English workshops

### About us

* Saigon Technology is a software outsourcing VN-based company with around 300 devs in HCM (headquarter) and Danang (branch). We focus on the US-UK market. Our devs are working directly with clients in English on a daily basis.

### Contact me:
Website: https://saigontechnology.com/
Email: nhien.lam@saigontechnology.com
Skype: https://join.skype.com/invite/haTZfmOx1YKo
Facebook: https://www.facebook.com/lam.chu.dieu.nhien/
LinkedIn: https://www.linkedin.com/in/lam-chu-dieu-nhien/
